lib/Test/Nginx/Util.pm

```diff
@@ -918,6 +918,12 @@ http {
     default_type text/plain;
     keepalive_timeout  68;
 
+    client_body_temp_path "$ServRoot/client_body_temp";
+    proxy_temp_path "$ServRoot/proxy_temp";
+    fastcgi_temp_path "$ServRoot/fastcgi_temp";
+    scgi_temp_path "$ServRoot/scgi_temp";
+    uwsgi_temp_path "$ServRoot/uwsgi_temp";
```

Comment:
> Your additions here will change the nginx.conf line numbers of the user configuration snippets, and thus will break existing tests checking line numbers of nginx.conf in some error messages.

I didn’t know that, I’ve moved them to the end of http block.

> Another thing is that you are assuming ngx_proxy, ngx_fastcgi, ngx_scgi, and ngx_uwsgi modules are always enabled in the nginx used in testing, which is apparently not the case. 

I know, but I don’t know how to enable them conditionally only when needed. Any idea?